source/test/TestApp/TestRunner.cpp

```diff
@@ -8,18 +8,16 @@
 #include "TestRunner.h"
 #include "TestUtils.h"
 
-using namespace PlayFab;
```

Comment:
nit: this seems like a readability regression to require adding PlayFab:: in most places (BUT, it makes sense if we want to differentiate the static GetTimePointNow from a potentially similarly named function here.

Reply:
Basically, we've been moving towards a stricter more verbose usage in all cases because of name conflicts in the past.
When it's trivial, I remove the most obvious and worst potential conflictors.

Reply:
I followed up offline about this but please be careful of this. There are dependencies on these `using` statements in some places that would be painful to deal with if this style was adopted more broadly in the SDK